src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx

```diff
@@ -12,35 +12,30 @@ type InterfaceDrawerDetailsTabProps = {
 };
 
 const InterfaceDrawerDetailsTab: FC<InterfaceDrawerDetailsTabProps> = ({ selectedInterface }) => {
-  const ports = selectedInterface.bridge?.port?.map((port) => port.name);
+  const { t } = useNMStateTranslation();
+  const ports = selectedInterface.bridge?.port?.map((port) => port.name)?.sort();
 
   return (
     <div>
-      {selectedInterface.ethtool?.feature && (
-        <div className="pf-u-mb-md">
-          <Title headingLevel="h4">{t('Features')}</Title>
-          <List isPlain>
-            {Object.keys(selectedInterface.ethtool?.feature)
-              .sort((a, b) => (a > b ? 1 : -1))
-              ?.map((feature) => (
-                <ListItem key={feature}>
-                  <Flex>
-                    <FlexItem>
-                      <Checkbox
-                        isDisabled
-                        id={`checkbox-${feature}`}
-                        value={feature}
-                        isChecked={selectedInterface.ethtool?.feature?.[feature]}
-                      />
-                    </FlexItem>
+      {selectedInterface.vlan && (
+        <div className="pf-u-mb-md" data-test="lldp-section">
```

Comment:
Do we really need this div? Maybe we could add `className` to `Flex` component below to style it, and this additional div wouldn't be needed. WDYT? Same for another div below in this file.

Reply:
Used `Stack` and `StackItem` with `hasGutter`